tests/components/modbus/conftest.py

```diff
@@ -46,7 +46,6 @@ async def setup_base_test(
     scan_interval,
 ):
     """Run setup device for given config."""
-
```

Comment:
This has nothing to do with changing constants (which I see as a good idea), the empty line is design,  please remove this change.